src/circshiftedarray.jl

```diff
@@ -27,10 +27,18 @@ julia> copy(s)
 """
 struct CircShiftedArray{T, N, S<:AbstractArray} <: AbstractArray{T, N}
     parent::S
+    # shift stores the circshift but is modified with mod (see below) to be more efficient
+    # in the getindex method
+    # negative shifts (left shift) are converted to positive (right shifts) with a larger
+    # shift number
```

Comment:
This comment should probably be part of the docstring of `CircShiftedArray`.

Reply:
I guess the comments (here and below) can probably be removed now that this information is in the docstring.